pkg/webui/console/containers/device-onboarding-form/device-type-form-section/device-type-manual-form-section/advanced-settings-section.js

```diff
@@ -284,6 +286,17 @@ const AdvancedSettingsSection = () => {
     }))
   }, [_default_ns_settings, defaultMacSettings, mayChangeToDefaultSettings, setValues])
 
+  // Register hidden fields so they don't get cleaned.
+  useEffect(() => {
+    const hiddenFields = [
+      'network_server_address',
+      'application_server_address',
+      'join_server_address',
+    ]
+    addToFieldRegistry(hiddenFields)
```

Comment:
```suggestion
    addToFieldRegistry(...hiddenFields)
```
You can spread the array. This way you don't have to flatten the field registry.